test/support/BaseSpec.scala

```diff
@@ -33,20 +33,22 @@ import play.api.mvc.Result
 import play.api.test.Helpers.{ contentAsString, defaultAwaitTimeout }
 import play.api.test.Injecting
 import play.twirl.api.Content
+import uk.gov.nationalarchives.omega.editorial.models.session.Session
 import uk.gov.nationalarchives.omega.editorial.models.{ Creator, PlaceOfDeposit, User }
-import uk.gov.nationalarchives.omega.editorial.services.ReferenceDataService
+import uk.gov.nationalarchives.omega.editorial.services.{ EditSetRecordService, EditSetService, ReferenceDataService }
 
 import scala.concurrent.Future
 
 class BaseSpec extends PlaySpec with GuiceOneAppPerSuite with Injecting with BeforeAndAfterEach {
 
   val user: User = User("dummy user")
-
   val testReferenceDataService: TestReferenceDataService = new TestReferenceDataService()
-
+  lazy val editSetRecordService: EditSetRecordService = new EditSetRecordService(testReferenceDataService)
```

Comment:
Can we use the injector to get instances of these classes, rather than constructing them manually? e.g.
```
lazy val editSetRecordService: EditSetRecordService = app.injector.instanceOf[EditSetRecordService]
```
Do we need references to these classes at the minute?

Reply:
Both services are used, yes.

I'll inject both.

Reply:
Changed.